tf/examples/bonsai_example.py

```diff
@@ -1,6 +1,9 @@
 import bonsaipreprocess
 import tensorflow as tf
 import numpy as np
+import sys
+sys.path.insert(0, '../')
```

Comment:
This should not be part of bonsai example. Can you please remove this? We will add this to `README.md` or something

Reply:
I know, I was testing it and forgot to remove it. Let it stay for our tests until you get steup.py up and running.